code/modules/food/drinkingglass/shaker.dm

```diff
@@ -12,19 +12,22 @@
 	var/lid_color = "black"
 
 /obj/item/weapon/reagent_containers/food/drinks/glass2/fitnessflask/Initialize()
-	..()
+	. = ..()
 	lid_color = pick("black", "red", "blue")
 	update_icon()
 
 /obj/item/weapon/reagent_containers/food/drinks/glass2/fitnessflask/update_icon()
 	..()
+	// And now set half the stuff back because our name shouldn't change
+	name = initial(name)
+	desc = initial(desc)
```

Comment:
IMO, this should just be proteinshake thing, since normal shakers that start out empty changing their name seems like reasonable thing?

Reply:
I pushed a fix for this, but
![image](https://user-images.githubusercontent.com/15028025/81026470-3703b200-8e48-11ea-842c-5c4204989fd9.png)